Themes/Catpuccin mocha.theme.json

```diff
@@ -0,0 +1,137 @@
+{
+    "name": "Catpuccin_Mocha",
+    "style": "Dark",
+    "author": [
+        {
+            "name": "Catpuccin Mocha",
+            "url": "https://catppuccin.com/"
+        }
```

Comment:
Referencing the original is a cool idea. Please feel free to also add yourself for credit where credit is due!

Reply:
Oh I don't know where should I add my nickname.